src/Mjolnir.ts

```diff
@@ -240,9 +243,11 @@ export class Mjolnir {
         const reportManager = new ReportManager(this);
         reportManager.on("report.new", this.handleReport);
         this.webapis = new WebAPIs(reportManager, this.ruleServer);
-
         // Setup join/leave listener
         this.roomJoins = new RoomMemberManager(this.client);
+        this.reportPoll = new ReportPoll(client, reportManager, async (_from: number) => {
```

Comment:
Ok, so I guess we will need to figure out how to express the choice between this and the report forwarding in the config.

Reply:
I also think that it is possible for you to test this, if you write your test such that the client connects directly to synapse and avoids nginx